src/pages/kimkia/Login/Login.js

```diff
@@ -0,0 +1,78 @@
+import './Login.scss';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+function Login() {
+  const navigate = useNavigate();
+  const [IdValue, setIdvalue] = useState('');
+  const [PWValue, setPWvalue] = useState('');
+  const [btnValue, setbtnvalue] = useState(true);
+
+  function goToMain() {
+    fetch('http://10.58.7.17:8000/users/login', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: IdValue,
+        password: PWValue,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => console.log('결과: ', result));
+  }
+
+  {
+  }
+  function handleIdInput(E) {
+    setIdvalue(E.target.value);
+  }
+
+  function handlePWInput(E) {
+    setPWvalue(E.target.value);
+  }
+
+  function active() {
+    if (IdValue.includes('@') && PWValue.length > 5) {
+      setbtnvalue(false);
+    } else {
+      setbtnvalue(true);
+    }
+  }
```

Comment:
```suggestion
  function active() {
    if (IdValue.includes('@') && PWValue.length > 5) setbtnvalue(!btnValue);
  }
```
해당 로직은 이렇게 리팩토링 할 수 있습니다